crates/nu-std/std/log/mod.nu

```diff
@@ -1,139 +1,128 @@
-export def log-ansi [] {
-    {
-        "CRITICAL": (ansi red_bold),
-        "ERROR": (ansi red),
-        "WARNING": (ansi yellow),
-        "INFO": (ansi default),
-        "DEBUG": (ansi default_dimmed)
-    }
+export const LOG_ANSI = {
+    "CRITICAL": (ansi red_bold),
+    "ERROR": (ansi red),
+    "WARNING": (ansi yellow),
+    "INFO": (ansi default),
+    "DEBUG": (ansi default_dimmed)
 }
```

Comment:
I'd suggest keeping the constant private and still export the commands:

1. to keep backwards compatibility: exported commands are module's API
2. to prevent the scope pollution, see below.

To expand on the second point, currently, when you import a module, its exported constants are always available in the 'global' scope:

![image](https://github.com/user-attachments/assets/a3efb02b-df73-47cc-bc3a-8a6763a3de64)

It's probably a bug and will hopefully be fixed one day.

```suggestion
const LOG_ANSI = {
    "CRITICAL": (ansi red_bold),
    "ERROR": (ansi red),
    "WARNING": (ansi yellow),
    "INFO": (ansi default),
    "DEBUG": (ansi default_dimmed)
}

export def log-ansi [] { $LOG_ANSI }
```

(ditto for other defs you removed).

Reply:
Yep, definitely. An immediate benefit of keeping the API is that the CI should pass without any modification to the tests.